flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/utils/TestingResourceManagerGateway.java

```diff
@@ -458,19 +458,6 @@ public CompletableFuture<TransientBlobKey> requestTaskManagerFileUploadByType(
         }
     }
 
-    @Override
-    public CompletableFuture<TransientBlobKey> requestTaskManagerFileUploadByName(
-            ResourceID taskManagerId, String fileName, Time timeout) {
-        final Function<Tuple2<ResourceID, String>, CompletableFuture<TransientBlobKey>> function =
-                requestTaskManagerFileUploadByNameFunction;
```

Comment:
Shall we remove the member variable `requestTaskManagerFileUploadByNameFunction` in `TestingResourceManagerGateway.java` at the same time?

Reply:
Thanks for your quick review!

Good catch, I have removed.